lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -94,6 +94,8 @@ defmodule RealtimeWeb.RealtimeChannel do
         db_conn: db_conn
       }
 
+      Tenants.track_active_tenant(tenant_id)
```

Comment:
where is the implementation of `track_active_tenant` now? You've changed it to `list_active_tenants` in the code above

Reply:
changed the spec only as it was not properly set 🤦 

it's still in `Realtime.Tenants`